yellowbrick/classifier/class_balance.py

```diff
@@ -25,6 +25,9 @@
 
 from .base import ClassificationScoreVisualizer
 
+from .base import ClassificationScoreVisualizer
```

Comment:
What is the Class Balance visualizer being altered?  Why is line 28 a duplicate of line 26?